codecov_auth/commands/owner/interactors/start_trial.py

```diff
@@ -12,7 +12,7 @@ def validate(self, owner: Owner):
 
     def _start_trial(self, owner: Owner) -> None:
         plan_service = PlanService(current_org=owner)
-        plan_service.start_trial()
+        plan_service.start_trial(current_owner=self.current_owner)
```

Comment:
Hmm this is confusing, just for my understanding, both `current_org` and `current_owner` are of `Owner` objects, but `current_org` is the GH organization that we want to start the trial on and `current_owner` is the authenticated GH user that is looking to start the trial of the `current_org` organization?

Reply:
yah you got it. We use current_user to refer to user object of the request user, and current_owner to refer to owner object of the request user. 

Are you confused over the owner model in general?

Reply:
Maybe something that can help read this better is changing `owner` to `current_org`? That way we better separate the concept of the owner/"user" that's doing the clicking vs the currently selected org, wdyt? If so, I'd make the change from the mutation call 2 levels up to keep parity 😬

Reply:
oh you mean the owner passed in start_trial? that could work